bot/cogs/events.py

```diff
@@ -117,9 +120,29 @@ async def on_guild_join(self, guild: Guild) -> None:
         except Exception:
             pass
 
-        await logchannel.send(
-            f"The bot has been added to **{guild.name}** , " f"We've reached our **{len(self.bot.guilds)}th** server! :champagne_glass: "
+        log_embed = Embed(
+            title=f"The bot has been added to {guild.name}",
+            color=Color.green(),
+            description=textwrap.dedent(
+                f"""
+                **We've reached our {len(self.bot.guilds)}th server!** :champagne_glass:
+                Guild Owner: **{guild.owner}** | **<@!{guild.owner_id}>**
+                Region: **{guild.region}**
+                Member Count: **{guild.member_count}**
+
+                Text channels: **{len(guild.text_channels)}**
+                Voice channels: **{len(guild.voice_channels)}**
+                AFK timeout: **{round(guild.afk_timeout / 60)}m | AFK channel: {guild.afk_channel}**
+
+                Created at: **{datetime.datetime.strftime(guild.created_at, "%A %d %B %Y at %H:%M")}**
+                File Size: **{round(guild.filesize_limit / 1048576)} MB**
+                Bitrate: **{round(guild.bitrate_limit / 1000)} kbps**
+                """
+            ),
```

Comment:
Why would we need this? The current embed is better imo

Reply:
we need it, later if the bot is abused, this can be used to find the server, and report it

Reply:
It's definitely kinda pointless but what's the harm in keeping it @e-xe ?

Reply:
So if someone abuses the bot from a server, will we go to the log channel, looking for the server info? Or can't we just send an email to discord with the server id?
And I see some unneeded information in there such as bit-rate, file size etc. All of them, actually, besides the server id and name and what server we are on.
```suggestion
        log_embed = Embed(
            title=f"Bot is invited to a server!",
            color=Color.green(),
            description=textwrap.dedent(
                f"""
                **We've reached our {len(self.bot.guilds)}th server!** :champagne_glass:
                Server name: {guild.name} | Id: {guild.id}
                """
            ),
```